website/pages/api-docs/jobs.mdx

```diff
@@ -30,6 +30,12 @@ The table below shows this endpoint's support for
 - `prefix` `(string: "")` - Specifies a string to filter jobs on based on
   an index prefix. This is specified as a query string parameter.
 
+- `all_namespaces` `(bool: false)` - Specifies whether to return the all
+  known jobs across all namespaces. When set to `true`, the list stubs will
```

Comment:
GitHub doesn't seem to let me make comments/recommendations on unchanged code :/, but I'm not keen on the idea that `all_namespaces` controls whether `ListStub.Namespace` is populated. this means, e.g., that code consuming the list of stubs has to know whether or not it was generated with `all_namespaces == true`. why not modify `Job.Stub()` to always set `JobListStub.Namespace`?

Reply:
I considered adding namespace on the stub - but worried about the increased size and we've been resisting adding more.  Also noticed that we already noticed `Job.Summary.Namespaces` is already always populated.

In one sense, it's already a redundant field but I added it anyway because I found `Job.Summary.Namespace` a bit odd considering namespace is primarily a field on job rather than the summary.

Reply:
cool. i didn't notice that namespace was already in `JobSummary`.

Reply:
FWIW - I was mostly explaining my rationale but I'm still open to adding Namespace field indiscriminately mostly for purity/beauty prospective if someone advocates for it :).  I just don't know how resistant we are to adding more fields here.

Reply:
no, i agree. i think the presence in `JobSummary` (presumably, to support `GET /job/:id:summary`) perhaps argues against putting it in the list stub at all.

i'd be curious to see what other folks think.